scripts/intfutil

```diff
@@ -53,7 +53,7 @@ def db_port_status_get(db, intf_name, status_type):
     if status is None:
         return "N/A"
 
-   if status_type == PORT_SPEED and status != "N/A":
+    if status_type == PORT_SPEED and status != "N/A":
```

Comment:
Hmm, no clue how that space got in there in the last second, this specific portion was tested several times.